ci/build_macos.sh

```
@@ -0,0 +1,58 @@
+#!/bin/bash -e
+# Generated with Claude Code (https://claude.com/claude-code)
+
+CI_DIR=$(readlink -f $(dirname $0))
+ROOT_DIR=$(readlink -f ${CI_DIR}/..)
+BUILD_SH_ARGS=$@
```

Shellcheck (0.11.0)

[warning] Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

(SC2124)